cla_backend/apps/cla_butler/tests/test_tasks.py

```diff
@@ -21,11 +21,14 @@ class TasksTestCase(TestCase):
     The tests should at this point continue to pass.
     """
 
+    delete_option_three_years = "three_years"
```

Comment:
Could we store these as constants in the tasks.py file and reference them in the both places?